doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -89,169 +70,245 @@ Workaround: Use the legacy adapter instead of the Elytron adapter. Enter the fol
-. Run the bulk reassignment. An error message appears telling you that some of the tasks could not be reassigned because they were not in the correct state.
-. Click *OK*.
-
-Expected result: The dialog box closes and the operation is attempted on all of the tasks. Results for particular tasks are presented separately as notifications.
+//approved
+. Log in to {CENTRAL} as two different users (A and B) from two different hosts or browsers.
+. As user A and as user B, open the same space.
+. As user A, import a project using an external Git repository URL.
 
-Actual result: The dialog box does not close. When you close it manually by clicking the *x* in the top right corner, and you check the states of the previously selected tasks, only the tasks processed before first failure are reassigned.
+Expected result: As user B, you can see the imported project in the space.
 
-Workaround: Before running the bulk action, filter the tasks and omit any task that will interrupt the reassignment.
+Actual result: As user B, you cannot see the imported project in the space.
 
-endif::[]
+Workaround: As user B, reload the space.
 
-.In the DMN designer, the DMN file validation operation does not report invalid operators [https://issues.jboss.org/browse/RHDM-1119[RHDM-1119]]
 
-Issue: In the DMN designer, if the DMN model contains an invalid operator (an operator that does not exist), for example `>>>>`, the validation operation does not report the invalid operator.
+//approved
+.In a high-availability authoring environment, project creation fails when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2475[RHPAM-2475]]
 
-Workaround: None.
+Issue: When you create a project, the project creation does not complete. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
 
-ifdef::DM[]
+Workaround: Create the project again.
 
 //approved
-.Users cannot test DMN context expression in test scenarios [https://issues.jboss.org/browse/RHDM-1116[RHDM-1116]]
+.In a high-availability authoring environment, asset creation is not completed when a {CENTRAL} node is down [https://issues.jboss.org/browse/RHPAM-2476[RHPAM-2476]]
+
+Issue: When you create an asset, the asset is not indexed. It is displayed in {CENTRAL} but you cannot use it. The cause is one {CENTRAL} node in a high-availability authoring environment going down.
+
+Workaround: In the *Project Explorer* view, open the asset and click *Save*.
+
+== DMN designer
 
-Issue: If you test DMN context expressions using the test scenario designer and the test fails, the corrected output has the wrong format.
+.When importing a data object from a Java class, a field of the Java Date type is not converted to the DMN date type [https://issues.redhat.com/browse/RHDM-1145[RHDM-1145]]
+
+Issue: When you use DMN designer to import a data object from a Java class and the class has a field of type Date, the resulting DMN field is not of a DMN date type.
```

Comment:
When you use the DMN designer to import a data object from a Java class and the class has a field of type `Date` the resulting DMN field does not have the DMN `date` type.

Reply:
I don't think I agree with removal of the comma but I remove it because I don't want to waste time on a minor point